src/App.js

```diff
@@ -1,26 +1,57 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component } from 'react'
+import Title from './Title'
+import jsondata from './content.json'
+import Foot from './foot'
+import Content from './content'
+import'./app.css'
+
+
+
+class App extends Component {
+
+    state = {
+        contentlist: jsondata.content[0],
+        showContent: true
+    }
+
+    rightClickChangeState = (contentindex) => {
+        const {contentlist} = this.state
+        fetch("./content.json")
```

Comment:
Try to use Redux to fetch data from a API or a JSON file.
https://redux.js.org/basics/usage-with-react